disttask/framework/storage/task_table.go

```diff
@@ -327,25 +296,25 @@ func (stm *SubTaskManager) HasSubtasksInStates(tidbID string, taskID int64, stat
 }
 
 // UpdateSubtaskState updates the subtask state.
-func (stm *SubTaskManager) UpdateSubtaskState(id int64, state string) error {
+func (stm *TaskManager) UpdateSubtaskState(id int64, state string) error {
 	stm.mu.Lock()
 	defer stm.mu.Unlock()
 
 	_, err := execSQL(stm.ctx, stm.se, "update mysql.tidb_background_subtask set state = %? where id = %?", state, id)
 	return err
 }
 
-// UpdateHeartbeat updates the heartbeat of the subtask.
-func (stm *SubTaskManager) UpdateHeartbeat(instanceID string, taskID int64, heartbeat time.Time) error {
+// UpdateSubtaskHeartbeat updates the heartbeat of the subtask.
+func (stm *TaskManager) UpdateSubtaskHeartbeat(instanceID string, taskID int64, heartbeat time.Time) error {
 	stm.mu.Lock()
 	defer stm.mu.Unlock()
 
 	_, err := execSQL(stm.ctx, stm.se, "update mysql.tidb_background_subtask set exec_expired = %? where exec_id = %? and task_key = %?", heartbeat.String(), instanceID, taskID)
 	return err
 }
 
-// DeleteTasks deletes the subtask of the given global task ID.
-func (stm *SubTaskManager) DeleteTasks(taskID int64) error {
+// DeleteSubtasksUnderGlobalTask deletes the subtask of the given global task ID.
+func (stm *TaskManager) DeleteSubtasksUnderGlobalTask(taskID int64) error {
```

Comment:
`DeleteSubtasksUnderGlobalTask` rename it like `DeleteSubtasksByTaskID` or `DeleteSubtasksUnderTask` ?